src/compiler/crystal/command.cr

```diff
@@ -139,9 +139,10 @@ class Crystal::Command
   rescue ex : OptionParser::Exception
     error ex.message
   rescue ex
+    ex.inspect_with_backtrace STDERR
```

Comment:
I guess this is a temporal change. Right?

Reply:
Yes, because things are broken right now, it breaks before this and I wanted to know why.

This all PR is temporary until you tell me "let's go with this", and then I can make things cleaner.